projects/adrv9009/profiles/tx_bw100_ir122p88_rx_bw100_or122p88_orx_bw100_or122p88_dc122p88/talise_config.c

```diff
@@ -209,7 +209,7 @@ taliseInit_t talInit = {
 	/* Digital Clock Settings */
 	.clocks =
 	{
-		.deviceClock_kHz = 122880,            /* CLKPLL and device reference clock frequency in kHz */
+		.deviceClock_kHz = 245760,            /* CLKPLL and device reference clock frequency in kHz */
```

Comment:
i think i should better try to come up with a fully working profile instead of just partially improving on this one.
will replace this commit with equivalent of https://github.com/analogdevicesinc/linux/commit/bd552e615421c58e96a71a7fe7014abd7578954a

Reply:
OK, we'll wait merging these changes until you do this.